R/utils.R

```diff
@@ -143,3 +143,28 @@ summarise_commit <- function(object) {
 
   return(out)
 }
+
+#' @title Load an application and instructions to run shiny.benchmark
+#'
+#' @param path A character vector of full path names
+#'
+#' @importFrom glue glue
+#' @export
+load_example <- function(path) {
+  # see if path exists
+  if (!file.exists(path))
```

Comment:
It might be worth adding another check and preventing the function from running in a non-empty directory.

Reply:
Great idea!